RecoParticleFlow/PFProducer/interface/PFEGammaFilters.h

```diff
@@ -56,24 +57,32 @@ class PFEGammaFilters {
   const unsigned int ele_missinghits_;
   const float ele_ecalDrivenHademPreselCut_;
   const float ele_maxElePtForOnlyMVAPresel_;
-  const float ele_maxNtracks, ele_maxHcalE, ele_maxTrackPOverEele, ele_maxE,
-    ele_maxEleHcalEOverEcalE, ele_maxEcalEOverPRes, ele_maxEeleOverPoutRes,
-    ele_maxHcalEOverP, ele_maxHcalEOverEcalE, ele_maxEcalEOverP_1,
-    ele_maxEcalEOverP_2, ele_maxEeleOverPout, ele_maxDPhiIN;
+  float ele_maxNtracks;
+  float ele_maxHcalE;
+  float ele_maxTrackPOverEele;
+  float ele_maxE;
```

Comment:
most other members already have an underscore.
If not too painful, please updated these as well.